inference-engine/src/vpu/graph_transformer/src/middleend/pass_manager.cpp

```diff
@@ -90,8 +90,9 @@ PassSet::Ptr PassManager::buildMiddleEnd() {
     //
     // Decompose swish layer to Sigmoid + Multiply
     //
-    ADD_PASS(decomposeSwish);
-    ADD_DUMP_PASS("decomposeSwish");
+    // TODO: remove this pass at all (issue 38626)
+    //ADD_PASS(decomposeSwish);
+    //ADD_DUMP_PASS("decomposeSwish");
```

Comment:
Is there really a need to keep these comments?

Reply:
Evgeny Latkin said that I can make a draft PR for now. And if everything works out, then delete unnecessary lines

Reply:
Well, make this PR as `draft`, please :)